AWSQueueManager.php

```diff
@@ -128,28 +134,39 @@ public function send($url, $message, $options=array())
      * with a limit of 10 per request. If $messageArray has more than 10 messages
      * then 2 requests will be triggered.
      *
+     * Important Note: If the SQS queue is FIFO type we MUST send the MessageGroupId also,
+     * otherwise the request will fail.
+     * @link https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_SendMessage.html
+     *
+     * Define if the SQS queue is a FIFO one via passing the `MessageGroupId` on $options param.
+     *
      * @param string $url          url of the queue to send message
      * @param string $messageArray message to send
      * @param array  $options      extra options for the message
      * @return boolean message was successful
      */
-    public function sendBatch($url, $messageArray, $options=array())
+    public function sendBatch($url, $messageArray, $options= [])
     {
-        $r=true;
-        foreach(array_chunk($messageArray,10) as $batch)
-        {
-            $messages=array();
-            foreach($batch as $i=>$message)
-            {
-                $messages[]=array(
+        $r = true;
+        foreach(array_chunk($messageArray,10) as $batch) {
+            $messages= [];
+            foreach($batch as $i=>$message) {
+
+                $messageData = [
                     'Id'          => $i,
                     'MessageBody' => (string)$message,
-                );
+                ];
+
+                if (isset($options['MessageGroupId'])) {
+                    $messageData['MessageGroupId'] = $options['MessageGroupId'];
+                }
+
+                $messages[]= $messageData;
             }
-            $result = $this->_sqs->sendMessageBatch(array_merge(array(
+            $result = $this->_sqs->sendMessageBatch(array_merge([
                 'QueueUrl' => $url,
                 'Entries'  => $messages,
-            ),$options));
+            ],$options));
```

Comment:
I'm thinking we might need to unset `MessageGroupId` from options here because it is operating at the message level, not the batch level.

Reply:
yes I was thinking the same, but I think will be ignored from the sqs API.
The `MessageGroupId` should be on Message level, but I will test both cases to be sure and will let you know.

Reply:
quick patch to test it on staging:
![Screenshot 2019-08-23 at 13 58 40](https://user-images.githubusercontent.com/9498242/63588438-2dceff80-c5af-11e9-8f26-3a62047d0614.png)

I was able to send the message:
![Screenshot 2019-08-23 at 14 05 17](https://user-images.githubusercontent.com/9498242/63588562-7dadc680-c5af-11e9-8d7f-bd69835d1664.png)

I have received also the needed email (a.k.a the Queue has been processed).
![Screenshot 2019-08-23 at 14 08 59](https://user-images.githubusercontent.com/9498242/63588662-bcdc1780-c5af-11e9-8613-ec995fc331d7.png)